setup.py

```diff
@@ -44,20 +44,20 @@ def package_files(package_dir, subdirectory):
                 'scripts/time_average.py', 'scripts/tophat_frfilter_run.py', 'scripts/model_calibration_run.py',
                 'scripts/time_chunk_from_baseline_chunks_run.py', 'scripts/chunk_files.py', 'scripts/transfer_flags.py',
                 'scripts/flag_all.py', 'scripts/throw_away_flagged_antennas.py', 'scripts/select_spw_ranges.py',
-                'scripts/multiply_gains.py', 'scripts/subselect.py'],
+                'scripts/multiply_gains.py', 'scripts/lstbin_simple.py', 'scripts/subselect.py'],
     'package_data': {'hera_cal': data_files},
     'install_requires': [
         'numpy>=1.10',
-        'scipy',
+        'scipy>=1.9.0',
         'h5py',
         'hdf5plugin',
         'astropy',
         'astropy-healpix',
-        'pyuvdata',
+        'pyuvdata @ git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata@cal-init-method',
```

Comment:
Again, let's make sure to get rid of these before we merge.

Reply:
Done, as above.